File: Results/DER/Translation/translation_output/no_test/granite/codenet/java/python/p03272.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```:
```
import java.util.*;

public class p03272 {
    public static void main(String[] args) throws Exception {
        Scanner sc=new Scanner(System.in);
        int N=sc.nextInt();
        int K=sc.nextInt();
        
        System.out.println(N-K+1);
    }
}

```


Answer:
```python
import sys

N, K = map(int, sys.stdin.readline().split())

print(N - K + 1)
```<|endoftext|>

File: Results/DER/Translation/translation_output/no_test/granite/codenet/python/java/p03272.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```:
```
n,i=map(int,input().split())
print(n+1-i)
```


Answer:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int i = scanner.nextInt();
        System.out.println(n + 1 - i);
    }
}
```<|endoftext|>
